File: src/main/java/com/jigumulmi/admin/place/dto/response/AdminS3PutPresignedUrlResponseDto.java
package com.jigumulmi.admin.place.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class AdminS3PutPresignedUrlResponseDto {

    @Schema(description = "presigned url")
    private String url;
    @Schema(description = "확장자 뺀 파일이름 (uuid)")
    private String filename;
}

File: admin/src/main/kotlin/com/jigumulmi/place/dto/response/AdminS3PutPresignedUrlResponseDto.kt
package com.jigumulmi.place.dto.response

import io.swagger.v3.oas.annotations.media.Schema

data class AdminS3PutPresignedUrlResponseDto (
    @Schema(description = "presigned url")
    private val url: String,

    @Schema(description = "확장자 뺀 파일이름 (uuid)")
    private val filename: String
)
